calculator.js

```diff
@@ -1,28 +1,74 @@
 
 const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
+  num1: 2,
+  num2: 4,
   operation: 'add',
 }
 // '3 + 5 = 8'
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+  let solution;
+  const num1 = promptInput.num1
+  const num2 = promptInput.num2
+
+  if (num1 == '' && num2 == '') {
```

Comment:
Use `===` instead of `==`

```suggestion
  if (num1 === '' && num2 === '') {
```